src/aky/aky_converter.c

```diff
@@ -428,10 +431,13 @@ int main(int argc, char **argv)
           snprintf(extra[i_extra], 21, "%"PRIu64, event.v_uint64[i_array]);
         }
         if(nb_extra) {
-          poti_user_PushState (specialPushState, timestamp, mpi_process, "STATE", value, nb_extra, extra);
+          poti_user_PushState_buffer (specialPushState, timestamp, mpi_process, "STATE", value, nb_extra, extra);
```

Comment:
Is there any reason for the `_buffer` variant? Such function remains undefined in the current PR.

Reply:
This variant is defined by this commit in my fork of Poti: https://github.com/Ezibenroc/poti/commit/d606bbd50656183328a699a6a432611233c8137e

I wanted to pass an arbitrary number of extra arguments, which was not possible with the variadic version of the function (or I don't know how).